cloudformation/cw-dashboard-substack.cfn.yaml

```diff
@@ -0,0 +1,223 @@
+{#- This variable is useful to decide where to put each graph/header section, it is #}
+{#- updated dynamically #}
+{%- set coord = {'x': 0, 'y': 0} %}
+{#- this macro allows to update the coord variable in order to put a new metric graph #}
+{#- If the graph we want to add is too big, we set x to 0 and y to be just after the last graph #}
+{%- macro update_coord(dx, dy) %}
+  {%- if coord.update({'x': (coord.x+dx)}) %} {%- endif %}
+  {%- if coord.x+dx > 24 %} {#- the longest width allowed is 24 #}
+    {%- if coord.update({'x': 0, 'y': coord.y+dy}) %} {%- endif %}
+  {%- endif %}
+{%- endmacro %}
+{#- this macro allows to update the coord variable in order to put a new header section #}
```

Comment:
Let's simplify these comments:
"Macro to calculate section header coordinates."

Reply:
Done